sourdough/resources/views/dailylog.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Daily Log</title>
+</head>
+<body>
+    <div class="daily-log">
+        <div id='myQWP'></div>
+        <div class="daily-log-form">
+            <div class="daily-log-form__title"> Add a New Entry </div>
+            <div class="daily-log-input">
+                <label for="date" class="daily-log-input__label">Date</label>
+                <input type="text" id="date" name="date" class="daily-log-input__field">
+            </div>
+            <div class="daily-log-input">
+                <label for="comment" class="daily-log-input__label">Comment</label>
+                <textarea id="comment" name="comment" class="daily-log-input__field"></textarea>
+            </div>
+            <div class="daily-log-input">
+                <div class="daily-log-input__label">Picture</div>
+                <label for="picture" class="daily-log-upload-label">upload</label>
+                <input type="file" id="picture" name="picture" onchange="onUpload(this)">
+                <div id="daily-log-upload-indicator"></div>
+            </div>
+            <div class="daily-log-form__submit" onclick="onSubmit()"> Submit </div>
+        </div>
+    </div>
+    <script>
+        init();
+    </script>
```

Comment:
Regarding the init of dailylog.html and dashboard.html, the respective scripts that are pulled in by the view.xml are in the <head/> of the web page. The init()s are in script tags in the <body/>. This ensures that the scripts are always loaded by the time page load gets to the body and calls the init(), doesn't it? Or am I misapprehending the way head and body scripts work? 
Putting 
`
        LABKEY.Utils.onReady({
            scope: this,
            scripts: ['/dailylog/script.js'],
                callback: () => {init();}
        });
`
in the head in this case doesn't work because the script is trying to find the div id myQWP in the body, but if the script init was in the head and didn't call `LABKEY.QueryWebPart` then I think that's the scenario in which to use LABKEY.Utils.onReady? Let me know what you think!